app/controllers/merchants_controller.rb

```diff
@@ -0,0 +1,17 @@
+class MerchantsController < ApplicationController
+  def index
+    response = Faraday.get('http://localhost:3000/api/v1/merchants')
+    body = response.body
+    @merchants = JSON.parse(body, symbolize_names: true)
+
+
+    @merchants = MerchantFacade.merchants
```

Comment:
My biggest piece of feedback is that this should be refactored out to a service/facade pattern which it looks like you've already started!